tests/test_processing_stack.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import time
+
+from pytest import mark
+
+APPLICATION_TAG_KEY = "ApplicationName"
+APPLICATION_TAG_VAL = "geospatial-data-lake"
+
+ENVIRONMENT_TAG_KEY = "EnvironmentType"
+ENVIRONMENT_TAG_VAL = os.environ.get("DEPLOY_ENV", "dev")
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def detect_compute_environment(batch_client):
```

Comment:
I feel like `get` would be more explicit than `detect` for these names. What do you think?

Reply:
Yes, I agree. Function names changed in a338c76dff1f809dc9c27b78fc3f38da7212af17